doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md

```diff
@@ -0,0 +1,353 @@
+
+Fill keyspace name. "kaa" is used in this example, because it's already created on a Sandbox machine.
+
+"logs_example" is used as the **Table name pattern**.
+
+![Keyspace configuration](attach/configuration-section-keyspace.png)
+
+The important part of configuration is **Column Mapping**:
+
+![Column mapping](attach/column-mapping.png)
+
+Other configuration:
+
+![Other configuration](attach/other-configuration.png)
+
+Now click **Add** button on the top of the screen to create and deploy appender.
+
+![Add button](attach/add-button.png)
+
+Verify that newly created appender has appeared in list.
+
+![Verify newly created appender](attach/verify-created-appender.png)
+
+Now run Data collection demo application:
+
+```bash
+java -jar DataCollectionDemo.jar
+2016-06-16 14:09:28,483 [main] INFO  o.k.k.d.d.DataCollectionDemo - Data collection demo started
+2016-06-16 14:09:29,795 [pool-2-thread-1] INFO  o.k.k.d.d.DataCollectionDemo - Kaa client started
+2016-06-16 14:09:29,798 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_0", "timeStamp": 1466075369795} sent
+2016-06-16 14:09:29,807 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_1", "timeStamp": 1466075369795} sent
+2016-06-16 14:09:29,807 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_2", "timeStamp": 1466075369795} sent
+2016-06-16 14:09:29,807 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_3", "timeStamp": 1466075369795} sent
+2016-06-16 14:09:29,807 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_4", "timeStamp": 1466075369795} sent
+2016-06-16 14:09:29,999 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [204 ms].
+2016-06-16 14:09:30,000 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [205 ms].
+2016-06-16 14:09:30,000 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [205 ms].
+2016-06-16 14:09:30,000 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [205 ms].
+2016-06-16 14:09:30,000 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [204 ms].
+2016-06-16 14:09:30,001 [pool-2-thread-1] INFO  o.k.k.d.d.DataCollectionDemo - Kaa client stopped
+2016-06-16 14:09:30,006 [main] INFO  o.k.k.d.d.DataCollectionDemo - Data collection demo stopped
+```
+
+Let's verify that our logs have been persisted in Cassandra. Go to Sandbox VM and run next command to connect Cassandra:
+
+```bash
+~$ cqlsh
+```
+
+Then
+
+```bash
+cqlsh> SELECT * FROM kaa.logs_example;
```

Comment:
Again, the 'cqlsh>' part is a prompt and should be removed. 
In general, the code part should contain only commands to enable Copy & Paste without manual editing.